snap/snapcraft.yaml

```diff
@@ -9,17 +9,17 @@ apps:
     server:
         daemon: simple
         command: rel/couchdb/bin/snap_run
-        plugs: [network-bind, process-control, mount-observe, browser-support]
+        plugs: [network-bind, process-control, mount-observe]
     couchdb:
         command: rel/couchdb/bin/snap_run
-        plugs: [network-bind]
+        plugs: [network-bind,browser-support]
 parts:
     couchdb: 
         plugin: make
-        #source: http://www-us.apache.org/dist/couchdb/source/2.0.0/apache-couchdb-2.0.0.tar.gz
-        source: https://github.com/apache/couchdb.git
-        source-type: git
-        source-tag: 2.1.0
+        source: http://www-us.apache.org/dist/couchdb/source/2.2.0/apache-couchdb-2.2.0.tar.gz
+        #source: https://github.com/apache/couchdb.git
+        #source-type: git
+        #source-tag: 2.2.0
         prepare: |
           ./configure --disable-docs
```

Comment:
If you switch to tarballs, you don't need to `--disable-docs` since the docs will already be built for you.